### DISCUSSION TITLE
Device Agent

### DISCUSSION THREAD

**Discussion by Linxford** [eneshecan/whatsapp-for-linux#218] [Q&A] [answered]
Great work, but upon installation and linking of the phone.......... it shows **safari Mac**....... how is that possible
am running it on linux

**Comment by xeco23** [comment, marked as answer]
Hi @wisdak. Please see #23. Unfortunately, there is not much we can do about it atm.

**Reply by Linxford** [reply]
alright........ believed in the near future it will be possible
